app/2.0/docs/devguide/dom-template.md

```diff
@@ -60,106 +59,183 @@ except for testing. For caveats about defining elements in the main document,
 see [main document definitions](registering-elements#main-document-definitions).
 {.alert .alert-info}
 
-### Specify a string template
+### Provide a template by defining a template property on the constructor {#templateobject}
 
-As an alternative to specifying the element's template in markup, you can specify a string template
-by creating a static `template` getter that returns a string.
+As an alternative to specifying the element's template in markup, you can define a static `template`
+property. The `template` property must be an `HTMLTemplateElement`.
 
-This getter is called _once_, when the first instance of the element is upgraded.
+Polymer provides an `html` function that can be applied to a template literal to produce an 
+`HTMLTemplateElement`. For example:
 
 ```js
 class MyElement extends Polymer.Element {
 
   static get template() {
-    return `<style>:host { color: blue; }</style>
+    return Polymer.html`<style>:host { color: blue; }</style>
        <h2>String template</h2>
-       <div>I've got a string template!</div>`
+       <div>I've got a string template!</div>`;
   }
 }
 customElements.define('my-element', MyElement);
 ```
 
-When using a string template, the element doesn't need to provide an `is` getter (however, the tag
-name still needs to  be  passed as the first argument to `customElements.define`).
+To enable HTML code highlighting in text editors with lit-html code highlighting functionality, you
+can declare a variable called `html` to hold the `Polymer.html` object, and return it from a `template` getter:
```

Comment:
Two details: `Polymer.html` is a function, and `html` here is a constant. 

"And return it from the template getter..." sounds like you're returning the function. Use it in the template getter, perhaps?